doc/README.md

```diff
@@ -1,22 +1,20 @@
-<!-- cspell: ignore tspconfig mgmt -->
+<!-- cspell: ignore tspconfig resourcemanager -->
 
 # Azure SDK for Rust Documentation
 
 This directory curates engineering guidance, design notes, and process documentation for contributors to the Azure SDK for Rust. Use these pages to understand repository conventions, service onboarding workflows, and supporting infrastructure.
 
 ## Documentation Map
 
--   [Development Guides](https://github.com/Azure/azure-sdk-for-rust/blob/main/doc/dev/README.md)
-    -   Management-plane generation walkthroughs and TypeSpec automation tips for building `azure_mgmt_*` crates.
-    -   Step-by-step instructions for scaffolding new clients, aligning with the Azure SDK design guidelines.
--   [Deprecation Process](https://github.com/Azure/azure-sdk-for-rust/blob/main/doc/deprecation-process.md)
-    -   Policies for deprecating APIs, handling breaking changes, and communicating timeline expectations.
+-   [Consumer Guides](https://github.com/Azure/azure-sdk-for-rust/blob/main/doc/): contains documentation for consumers of SDKs
+-   [Development Guides](https://github.com/Azure/azure-sdk-for-rust/blob/main/doc/dev/README.md): contains advanced documentation for developers of SDKs (not consumers of SDKs)
+
+### Consumer Guides
```

Comment:
When we reorganize, I'd put this first before dev docs, but this is fine for now.